### PR TITLE
Added more instructions to node.js page

### PR DESCRIPTION
**Pull request by Xpert87**
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "license": "MIT",
   "dependencies": {
     "next": "^12.1.0",
-    "nextra": "^1.1.0",
```
Comment (QuickFinesse) [inline package.json]: We should probably keep the nextra version more up to date, try running `npm remove nextra` and then `npm install nextra`

**Review by QuickFinesse — CHANGES_REQUESTED** [review]
Upgrade nextra package and you're good to go, thanks for the contribution!

File: package.json (hunk 1 above)
Comment (Xpert87) [reply package.json]: Updated nextra version to 1.1.0. Don't know what happened with that

**Review by QuickFinesse — APPROVED** [review]
Looks good!

**Pull request merged**